### PR TITLE
[Wait for #2184][CAPI/Unittest] improve the possibility of a memory leak 

### PR DESCRIPTION
**Pull request by helloahn**
free the PATHs before checking `ASSERT`

it resolves https://github.com/nnsuite/nnstreamer/pull/2184#discussion_r395413523

Signed-off-by: Hyoung Joo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/tizen_capi/unittest_tizen_capi.cc [hunk 1]
```diff
@@ -1039,7 +1040,7 @@ TEST (nnstreamer_capi_switch, dummy_01)
   EXPECT_EQ (status, ML_ERROR_NONE);
 
   if (node_list) {
-    gchar *name = NULL;
+  gchar *name = NULL;
```
Comment (wooksong) [inline tests/tizen_capi/unittest_tizen_capi.cc]: Please check indentation here.

File: tests/tizen_capi/unittest_tizen_capi.cc [hunk 2]
```diff
@@ -1133,7 +1134,7 @@ TEST (nnstreamer_capi_switch, dummy_02)
...
```
Comment (wooksong) [inline tests/tizen_capi/unittest_tizen_capi.cc]: ditto

**Comment by myungjoo** [comment]
Do you really need to free things before calling assert?
Recent versions of coverity complain this, but I think Coverity is wrong with it.

File: tests/tizen_capi/unittest_tizen_capi.cc [hunk 3]
```diff
@@ -1836,14 +1838,16 @@ TEST (nnstreamer_capi_singleshot, invoke_02)
   int status;
 
   const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH");
-  gchar *test_model;
+  gchar *tmp, test_model[MAX_PATH_LEN];
```
Comment (wooksong) [inline tests/tizen_capi/unittest_tizen_capi.cc]: I knew this means that
```c
gchar (*tmp), (test_model[MAX_PATH_LEN]);
```
However, for the readability, please consider
```c
gchar test_model[MAX_PATH_LEN];
gchar *tmp;
```

> It’s also important to comment data, whether they are basic types or derived types. To this end, use just one data declaration per line (no commas for multiple data declarations). This leaves you room for a small comment on each item, explaining its use. 
from https://www.kernel.org/doc/html/v4.10/process/coding-style.html

I think this aims not just room for comment but also the readability of code.

**Pull request closed without merging**